RecoMET/METFilters/plugins/HcalStripHaloFilter.cc

```diff
@@ -0,0 +1,57 @@
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDFilter.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "DataFormats/METReco/interface/BeamHaloSummary.h"
+
+class HcalStripHaloFilter : public edm::EDFilter {
```

Comment:
new legacy modules are not allowed. Module type should be thread-aware 
https://twiki.cern.ch/twiki/bin/view/CMSPublic/FWMultithreadedFrameworkModuleTypes
edm::global::EDFilter would be a better choice.